app/src/main/java/co/develhope/meteoapp/fragments/SearchFragment.kt

```diff
@@ -5,21 +5,37 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import co.develhope.meteoapp.R
-
-
+import co.develhope.meteoapp.adapter.RecentSearchAdapter
+import co.develhope.meteoapp.data.DataSet
+import co.develhope.meteoapp.model.RecentSearch
 
 
 class SearchFragment : Fragment() {
 
+    private lateinit var recentSearchList  :  List<RecentSearch>
```

Comment:
Non sono necessarie, la lista la puoi recuperare direttamente quando devi inizializzare l'adapter. Idem per la recyclerview, soprattuto se non la riusi in altri punti del fragment